samples/Demo Data/Assets/Plugins/Android/mainTemplate.gradle

```
@@ -18,6 +18,7 @@ dependencies {
 
 // Android Resolver Exclusions Start
 android {
+	namespace "com.unity3d.player"
```

```suggestion
  namespace "com.unity3d.player"
```

should this value be like `com.datadog.unity.xxx`?

---

This got inserted by Unity.... I'll double check it it's actually a setting somewhere that I didn't override properly.

---

I think this is correct because the initial activity is in the com.unity3d.player namespace.